### PR TITLE
Added web/8.4 version

### PR DESCRIPTION
**Pull request by tgrousseau**
- Added 8.4 version for web 
- Moved current docs to 8.3 version
- Added a Changelog page for web on both 8.3 & 8.4 (both up to date)
- Updated Customization page on both 8.3 & 8.4 : 
  - Moved to a category
  - Title changed to "Customization and optional features"
  - Updated old page title to "window.miam"
  - Window.miam page was redesigned and is up to date
  - Added a new page "Receiving baskets from affiliated websites" on both 8.3 & 8.4
  - On 8.4: Added a new page "Scanning a groceries list"
- Updated "Features" page title to "New features"

### CODE REVIEW

File: docs/web/customization/affiliated-websites.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  :::
+
+
+## Set force POS callback
+
+On the affiliated website, Mealz needs to ask to the user which store they wants to shop in, so we can know which product to present them.
+
+This means that, when they are redirected to your website, they expect to be on the store they chose earlier.
+
+**Which means that we need you to provide us a way to force the store !**
```
Comment (diarmuidmcg) [inline docs/web/customization/affiliated-websites.md:42]:
```suggestion
**Which means that we need you to provide us a way to force the user to select a store !**
```

File: docs/web/customization/affiliated-websites.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+```javascript
+// Example Setup
+export class Mealz {
+  constructor() {
+    window.miam.hook.setForcePosCallback(this.forceStoreCallback)
+  }
+
+  forceStoreCallback(storeId) => {
+    // Force the store to the store with the ID passed in param
```
Comment (diarmuidmcg) [inline docs/web/customization/affiliated-websites.md:61]:
```suggestion
    // Force the user to select a store with the ID passed in param
```

**Review by diarmuidmcg — APPROVED** [review]
juste des petits soucis en Anglais mais en gros c'est bon

File: docs/web/customization/affiliated-websites.md (hunk 1 above)
Comment (tgrousseau) [reply docs/web/customization/affiliated-websites.md:42]: Non non, c'est bien "force the store", je parle de l'intégration Marmiton pour les enseignes, il faut pouvoir forcer le magasin à celui qui a été choisi sur Marmiton

File: docs/web/customization/affiliated-websites.md (hunk 2 above)
Comment (tgrousseau) [reply docs/web/customization/affiliated-websites.md:61]: Pareil

File: docs/web/customization/window-miam.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    
+    Outside of this usage we don't recommand using `basket.reset()` except to quickly empty Mealz's basket for testing purposes
+  :::
+- `updatePricebook: (pricebookName: string) => Observable<void>`: If you have different price tables (or pricebooks) for the same point of sale, you can call this method to let Mealz know which pricebook you are using. For example, prices could be different depending on if the user chose home delivery or drive-in
+  :::info
+    If you have different pricebooks, they need to have been sent to our API so you can change them with `basket.updatePricebook()`. The pricebookName should be the same that the one sent to our API
+  :::
+- `recipeCount: () => Observable<number>`: A BehaviorSubject that emits the current number of recipes in Mealz' Basket once (it waits for the Basket to be initialized to emit).
+- `openPreview: () => void`: Opens the recipe-modal in basket preview mode to display the recipes currently in the basket (Same action as when clicking on the FAB in the recipe-catalog)
```
[the remark this answers is not in the document]
Comment (tgrousseau) [reply docs/web/customization/window-miam.md:42]: Floating Action Button

**Review by diarmuidmcg — APPROVED** [review]

**Pull request merged**